dc-cudami-editor/src/components/forms/RenderingTemplateForm.tsx

```diff
@@ -0,0 +1,220 @@
+import '../../polyfills'
+
+import omit from 'lodash-es/omit'
+import {useEffect, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {
+  Card,
+  CardBody,
+  Col,
+  Form,
+  Nav,
+  Row,
+  TabContent,
+  TabPane,
+} from 'reactstrap'
+import {useContext} from 'use-context-selector'
+
+import {getByUuid, save, typeToEndpointMapping, update} from '../../api'
+import {
+  addLanguage,
+  removeLanguage,
+  setActiveLanguage,
+  setFeedbackMessage,
+  toggleDialog,
+} from '../../state/actions'
+import {
+  getActiveLanguage,
+  getAvailableLanguages,
+  getDialogsOpen,
+  getExistingLanguages,
+  getFeedbackMessage,
+} from '../../state/selectors'
+import {Context} from '../../state/Store'
+import {DialogNames, RenderingTemplate} from '../../types'
+import AddLanguageDialog from '../dialogs/AddLanguageDialog'
+import RemoveLanguageDialog from '../dialogs/RemoveLanguageDialog'
+import FeedbackMessage from '../FeedbackMessage'
+import InputWithLabel from '../InputWithLabel'
+import LanguageAdder from '../LanguageAdder'
+import LanguageTab from '../LanguageTab'
+import {cleanLocalizedText} from '../utils'
+import ActionButtons from './ActionButtons'
+
+interface Props {
+  uuid: string
+}
+
+const submitData = async (
+  context: string,
+  data: RenderingTemplate,
+  type: string,
+  uuid: string,
```

Comment:
Kann man die nicht aus `data` holen?

Reply:
Beim `save` hab ich noch keine UUID :wink:

Reply:
Eben. Dann ist die in `data` ja auch nicht vorhanden. Aber beim Update sollte doch eine da sein. Dann muss man die ja nicht extra hier reinreichen. Mann könnte in Z. 54ff. ja auf `data?.uuid` testen, ode?

Reply:
Hab ich entsprechend überarbeitet :+1: